tests/base/utilities_01.cc

```diff
@@ -65,6 +65,18 @@ void test ()
                  ExcInternalError());
   }
 
+  {
+    const char *p = "alpha;; beta;; gamma ";
+    AssertThrow (Utilities::split_string_list (p, ";;").size() == 3,
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[0] == "alpha",
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[1] == "beta",
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[2] == "gamma",
+                 ExcInternalError());
```

Comment:
Reminds me that we might consider cleaning up all the places that unnecessarily use `AssertThrow` and `Assert` in the testsuite at some point. A simple `if (...) deallog << ...;` is much more reliable.